ws-nextjs-app/pages/[service]/live/[id]/Stream/index.tsx

```diff
@@ -21,6 +21,22 @@ const Stream = ({
     },
   } = useContext(ServiceContext);
 
+  const [hasShareApi, setHasShareApi] = useState(false);
+  const [hashValue, setHashValue] = useState('');
```

Comment:
Is `hashValue` used? Would the window object not have the hash already on it if its been shared?

Reply:
I think this is set to guarantee that the shared post title gains focus correctly, @emilysaffron?

Reply:
if i'm honest i dont remember why i added this 😭 because thinking about it i do agree with what aaron is saying in that if someone is coming from a shared link the hash value would already be anchored to? Like we shouldnt need the `window.location.href = hash` maybe worth removing it and just seeing what breaks if anything. Likely the focus thing like you say harvey

Reply:
Behaviour with the current hash code:

https://github.com/bbc/simorgh/assets/4706195/7c3e9a23-7758-47fa-b879-f6f3275ab445

Behaviour without the current hash code:

https://github.com/bbc/simorgh/assets/4706195/4ed116e6-41bc-4ecc-a1a4-cb224f577277

Reply:
Strange behaviour. Wonder why the second doesn't cause focus to occur. I thought the browser would handle that.

You could drop the `setState` stuff and do something like this to force focus, but I'm not sure of the a11y implications there:
```
document.getElementById(window.location.hash.substring(1))?.focus();
```

Reply:
Just a little concerned about setting the `window.location.href`, as it may have implications on browser history.